documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -108,4 +108,4 @@ You may wish to refer to [Cloudflare's documentation for deploying a SvelteKit s
 
 ### Accessing the file system
 
-You can't access the file system through methods like `fs.readFileSync` in Serverless/Edge environments. If you need to access files that way, do that during building the app through [prerendering](https://kit.svelte.dev/docs/page-options#prerender). If you have a blog for example and don't want to manage your content through a CMS, then you need to prerender the content (or prerender the endpoint from which you get it) and redeploy your blog everytime you add new content.
+You can't use `fs` in workers — you must [prerender](https://kit.svelte.dev/docs/page-options#prerender) the routes in question.
```

Comment:
it would seem like "workers" need to be capitalized as the product name or is this referring to web workers (https://vitejs.dev/config/worker-options.html) in which case we might need to add "web" there

is it confusing to have this message for Cloudflare Pages? I'm not that familiar with whether you might be able to use workers in a Pages app

```suggestion
You can't use `fs` in Workers — you must use Pages or [prerender](https://kit.svelte.dev/docs/page-options#prerender) the routes in question.
```

Reply:
Cloudflare Pages _is_ workers. We can change 'workers' to 'Cloudflare Workers' if nec

Reply:
```suggestion
You can't use `fs` in Cloudflare Workers — you must [prerender](https://kit.svelte.dev/docs/page-options#prerender) the routes in question.
```

Reply:
While Cloudflare Pages is Workers under the covers, there are things that are different between the environments. Maybe we just say "Cloudflare runtime" or something to avoid any potential for confusion?